docs/demos/manipulation.md

```diff
@@ -50,12 +50,20 @@ This demo showcases the capabilities of RAI in performing manipulation tasks usi
    ros2 launch examples/manipulation-demo.launch.py game_launcher:=path/to/RAIManipulationDemo.GameLauncher
    ```
 
-2. In the second terminal, run the interactive prompt:
+2. In the second terminal, run the cli prompt or streamlit interface:
 
    ```shell
    python examples/manipulation-demo.py
    ```
 
+   This script will prompt you to enter a natural language command in the terminal. It also serves as an example of how to use the RAI API for your own applications.
+
+   If you just want to try the demo on its own with a more user-friendly interface, you can use the Streamlit version:
+
+   ```shell
+   streamlit run examples/manipulation-demo-streamlit.py
```

Comment:
I would make streamlit the default. We can keep the terminal one for now, but I would prioritize streamlit.

Reply:
I updated the docs to address that :+1: